examples/dt/adoption/rhv/README.md

```diff
@@ -0,0 +1,61 @@
+# RHEV base adoption Topology
```

Comment:
nit" I believe it is RHV and not RHEV. `s/RHEV/RHV/g`
`RHV based Adoption Topology`